apps/web-connect/.gitignore

```diff
@@ -11,6 +11,7 @@ node_modules
 dist
 dist-ssr
 *.local
+.env
```

Comment:
Yes, I've now added a sample.env for reference.

Reply:
It does not make sense to have `VITE_APP_ENV` in an .env file. We need to set it on build time, web-connect is statically build, so there is no runtime making use of the .env file. Then, if its in a file we would have to write that file in the build pipeline but we would rather just incejt the environment variable at build step.

Please let me know if that makes sense, then remove the sample.env please.

It would make sense to write a readme that explains why and when this env variable is important to be added and what its used for.

Reply:
Thank you for the feedback. 

How should this below work in a local development environment for the app to know if it is in production or staging?

`const environment = process.env.VITE_APP_ENV  === 'development' ? 'development' : 'production';`

I was going off our previous Slack convo where you gave me this variable name to use for this purpose. 

![Vite](https://github.com/user-attachments/assets/8035b400-766c-4f02-b8db-1d282e6a7268)

Reply:
Per our Slack conversation. .env file added for local development. 

ReadMe updated to inform devs.